server/util/file.py

```diff
@@ -71,6 +71,17 @@ def retrieve_file(storage_path: str) -> BinaryIO:
         return open(storage_path, "rb")
 
 
+def delete_file(storage_path: str):
+    if config.FILE_UPLOAD_STORAGE_PATH.startswith("s3://"):
+        assert storage_path.startswith("s3://")
+        parsed_path = urlparse(storage_path)
+        bucket_name = parsed_path.netloc
+        key = parsed_path.path[1:]
+        s3().delete_object(bucket_name, key)
```

Comment:
if this fails, then we'll be left with an election where some files have been deleted, right?

Probably okay, though may want to improve this at some point.

Reply:
Yes, that's true. Given that the goal is deleting everything, a failure that only partially completes feels like a not-so-bad outcome, so I'm also ok with leaving it for now